lib/facebook/messenger/bot.rb

```diff
@@ -29,147 +30,147 @@ module Bot
         message_echo
         payment
         policy_enforcement
+        standby
       ].freeze
 
       BROADCAST_MESSAGE_REGULAR = 'REGULAR'.freeze
       BROADCAST_MESSAGE_SILENT  = 'SILENT_PUSH'.freeze
       BROADCAST_MESSAGE_NO_PUSH = 'NO_PUSH'.freeze
 
-      class << self
```

Comment:
While I'm normally in favor of these refactors, I'd advise against large ones like this because as we diverge more and more from the source repo, it makes it harder and harder to pull in their changes into this repo because the diffs are HUGE and have completely unrelated code changes in them.

This entire PR should be smaller, and more focused, with less refactors.